mne/viz/evoked.py

```diff
@@ -33,6 +33,12 @@
                                _auto_topomap_coords)
 
 
+# Valid data types ordered for consistency
+_VALID_CHANNEL_TYPES = ['eeg', 'grad', 'mag', 'seeg', 'eog', 'ecg', 'emg',
+                        'dipole', 'gof', 'bio', 'ecog', 'hbo', 'hbr',
+                        'misc']
```

Comment:
This should probably live in `..io.pick` instead (we'll want to add entries to it if we add new channel types)